docs/source/conf.py

```diff
@@ -423,3 +423,7 @@ def find_source():
 
 """
 coverage_skip_undoc_in_source = True
+
+suppress_warnings = [
+    'epub.unknown_project_files'
```

Comment:
```suggestion
#    'epub.unknown_project_files'
```
just to try to find the freezing

Reply:
the build seems fine now, shall we remove the suppress warning now?